src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -169,8 +169,18 @@ void SimulatedTestFixture::runTest(
 
     // Tick one frame to aid with visualization
     bool validation_functions_done = tickTest(simulation_time_step, ai_time_step, world);
+
```

Comment:
we can use this `tickTest` to initialize `max_tick_duration` and `min_tick_duration` as well as update `total_tick_duration` and `tick_count`

Reply:
Could you elaborate on your recommendation? How would the values be updated inside `tickTest` and then printed in `runTest`? Would the values be passed as an argument by reference to `tickTest` when it's called so it can be updated there? If that's the case, what would be the benefit of doing such

Reply:
what I mean is that instead of setting min_tick_duration to 10000.0, we can set min_tick_duration to the time that the tickTest on line 171 takes to run

Reply:
I see, that makes sense! I've updated it